src/Whoops/Resources/css/whoops.base.css

```diff
@@ -157,6 +157,14 @@ header {
     }
 
     .frame-index {
+      float: left;
```

Comment:
I think inline-blocks are only going to work if you specify a width for both blocks (I've just tried it). But then you have to do it in percents because the width of the left-panel depends on the screen size.
What's wrong with a float:left here?